shotover-proxy/tests/runner/hotreload_int_tests.rs

```diff
@@ -1,10 +1,81 @@
 use crate::shotover_process;
+use redis::{Client, Commands};
+use std::time::Duration;
+use test_helpers::docker_compose::docker_compose;
+use tokio::time::sleep;
+
 #[tokio::test]
-async fn test_hotreload_cli_flag_accepted() {
-    let shotover_process = shotover_process("tests/test-configs/null-valkey/topology.yaml")
+async fn test_hotreload_basic_redis_connection() {
+    let _compose = docker_compose("tests/transforms/docker-compose-moto.yaml");
```

Comment:
this is the wrong docker compose file, its for a specific test that needs the "moto" aws mock service, which is unrelated to this test.
Usually we create a new `docker-compose.yaml` file for each test (although sometimes we share them)
In this case we should create a new `docker-compose.yaml` and put it in the same folder as our `topology.yaml` files.

On that note, looking at our `topology.yaml` files we should put them in the same folder, probably create a directory `test-configs/hotreload`.